jetty-xml/src/main/java/org/eclipse/jetty/xml/XmlConfiguration.java

```diff
@@ -1799,10 +1801,18 @@ else if (arg.toLowerCase(Locale.ENGLISH).endsWith(".properties"))
                 return null;
             });
         }
-        catch (Error | Exception e)
+        catch (Error e)
         {
             LOG.warn(e);
             throw e;
         }
+        catch (Exception e)
+        {
+            if (!TerminateStartupException.isTerminateStartup(e)) 
```

Comment:
I really don't like XmlConfiguration having knowledge of the termination exception.... I guess I really still don't like the terminate exception.